distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Enso_File.enso

```diff
@@ -138,12 +139,123 @@ type Enso_File
+        asset.metadata.description.if_nothing ""
+
+    ## GROUP Metadata
+       ICON data_output
+       Sets the description associated with the file in Enso Cloud.
+
+       Arguments:
+       - description: The new description to set.
+
+       Returns:
+       - The file on which it was called.
+    set_description : Text -> Enso_File
+    set_description self (description : Text) -> Enso_File =
+        Context.Output.if_enabled disabled_message="Cannot set a description when writing is disabled. Press the Write button ▶ to perform the operation." panic=False <|
+            update_asset_description self description . if_not_error self
+
+    ## GROUP Metadata
+       ICON metadata
+       Gets the labels associated with the file in Enso Cloud.
+    labels : Vector Text
+    labels self -> Vector Text =
+        asset = Existing_Enso_Asset.get_asset_reference_for self want_metadata=True
+        asset.metadata.labels
+
+    ## GROUP Metadata
+       ICON data_output
+       Adds a label to the file in Enso Cloud.
+
+       If the label with the given name did not exist before, it is created with
+       a random color associated with it. If you want to set a specific color,
+       use `Enso_File.create_label` before calling this method.
+
+       Arguments:
+       - label: The name of the label to add to the file.
+
+       Returns:
+       - The file on which it was called.
+
+       ? Race Conditions
+
+         Adding labels is not atomic. If two processes are modifying labels of
+         the same asset at the same time, some changes may be lost.
+    add_label : Text -> Enso_File
+    add_label self (label : Text) -> Enso_File =
+        Context.Output.if_enabled disabled_message="Cannot add a label when writing is disabled. Press the Write button ▶ to perform the operation." panic=False <|
+            labels = self.labels
+            if labels.contains label then self else
+                ensure_tags_exist [label] . if_not_error <|
+                    new_labels = labels+[label]
+                    update_asset_labels self new_labels . if_not_error self
+
+    ## GROUP Metadata
+       ICON data_output
+       Removes a label from the file in Enso Cloud.
+
+       Arguments:
+       - label: The name of the label to remove from the file.
+
+       Returns:
+       - A boolean value indicating if the file had the label before the operation.
+
+       ? Race Conditions
+
+         Removing labels is not atomic. If two processes are modifying labels of
+         the same asset at the same time, some changes may be lost.
+    remove_label : Text -> Boolean
+    remove_label self (label : Text) -> Boolean =
+        Context.Output.if_enabled disabled_message="Cannot remove a label when writing is disabled. Press the Write button ▶ to perform the operation." panic=False <|
+            labels = self.labels
+            if labels.contains label . not then False else
+                new_labels = labels.filter l-> l != label
```

Comment:
It would be nice to have this as `Vector.remove_element`.

Reply:
Indeed, but adding it I'd have to add unit tests for it and I don't want to postpone this PR.

Let's add it in a separate PR when we see it used more, or see a need for it for end-users.